AdyenComponents/SEPA Direct Debit/SEPADirectDebitComponent.swift

```diff
@@ -142,3 +145,17 @@ extension SEPADirectDebitComponent: TrackableComponent {}
 
 @_spi(AdyenInternal)
 extension SEPADirectDebitComponent: ViewControllerDelegate {}
+
+// MARK: - SubmitCustomizable
+
+extension SEPADirectDebitComponent: SubmitCustomizable {
+
+    public func submit() {
+        guard !configuration.showsSubmitButton else {
+            AdyenAssertion.assertionFailure(message: "Default submit button must be hidden in order to call submit.")
```

Comment:
It always is the same error message right? Maybe we can move it to an extension of `SubmitCustomizable` something along the lines of `func triggerMisconfiguredSubmitButtonAssertion()` or something shorter :D

Reply:
Yes, it's always the same check but different components have a different set up for the showsSubmitButton property. The `SubmitCustomizable` does not know about the property.